packages/core/src/components/progress/progress.story.tsx

```diff
@@ -0,0 +1,37 @@
+import { c, classy, m, ProgressProps } from '@onfido/castor';
+import { html } from '../../../../../docs';
+
+export const Progress = ({
+  size,
+  value,
+  max = 100,
+  hideLabel,
+  children,
+  ...props
+}: ProgressProps) =>
+  html('div', {
+    ...props,
+    class: classy(c('progress'), m(size)),
+    role: 'progressbar',
+    'aria-valuenow': String(value),
+    'aria-valuemin': String(0),
+    'aria-valuemax': String(max),
+    'aria-valuetext': !hideLabel && children ? children : undefined,
```

Comment:
💅🏻  [nitpick]

```suggestion
    'aria-valuetext': (!hideLabel && children) || undefined,
```

Reply:
This is changed now to match the new implementation so it's not exactly as you have suggested.